lib/i2cscan/i2cscan.cpp

```diff
@@ -74,7 +74,7 @@ namespace I2CSCAN
             }
             else if (error == 4)
             {
-                Serial.print("[ERR] I2C (@ " + portMap[i] + " : " + portMap[j] + "): ");
+                Serial.print("[ERR] I2C (@ " + portMap[i] + "(" + portArray[i] + ") : " + portMap[j] + "(" + portArray[j] + ")): ");
```

Comment:
```suggestion
                Serial.printf("[ERR] I2C (@ %s (%s) : %s (%s)): ", portMap[i], portArray[i], portMap[j], portArray[j]);
```

Reply:
Should be `%s %d %s %d`. portArray is int, portMap is string.

Reply:
```
                Serial.printf("[ERR] I2C (@ %s(%d) : %s(%d)): Unknow error at address 0x%02x\n", 
                                portMap[i].c_str(), portArray[i], portMap[j].c_str(), portArray[j], address);
```

Reply:
That's not a suggestion...

Reply:
I've edited my suggestion